projects/wave-core-new/src/lib/backend/backend.service.ts

```diff
@@ -197,6 +198,19 @@ export class BackendService {
         return new HttpHeaders()
             .set('Authorization', `Bearer ${sessionId}`);
     }
+
+    // TODO: turn into paginated data source
+    getDataSetStream(sessionId: UUID): Observable<Array<DataSet>> {
```

Comment:
```suggestion
    getDataSets(sessionId: UUID): Observable<Array<DataSet>> {
```

You actually get one response which happens to be also an observable like other streams. But as a naming convention, I would only have plain requests here.

Reply:
d41c9c697d9f1cfc4b99f1989c59dee284d3d0a9